server/server.go

```diff
@@ -52,7 +52,9 @@ type httpClientFactory struct {
 }
 
 func (f httpClientFactory) MakeForSink(sinkName, monitorId string, opts, tags map[string]string) (*http.Client, error) {
-	client := &http.Client{}
+	client := &http.Client{
+		Timeout: 5 * time.Second,
```

Comment:
We shouldn't hard-code this because we can't know what's valid for all users. Instead, you'll want to implement https://github.com/cashapp/blip/blob/main/blip.go#L108

Reply:
Good catch. It makes sense to implement client specific httpclient.